apps/docs/content/components/CTABanner.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: CTABanner
+status: Experimental
+figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=539%3A14285&t=RJEWZhVUtrRQ0pKh-0'
+source: https://github.com/primer/brand/tree/main/src/CTABanner/CTABanner.tsx
+storybook: '/brand/storybook/?path=/docs/components-ctabanner--playground'
+description: Use CTABanner to create call to action content in a separated section.
```

Comment:
How about this..

"Use CTABanner to draw attention to or create urgency around a user action"

cc. @ajashams @tobiasahlin @danielguillan for their suggestions also.

Reply:
Sounds much better to me.